services/brig/src/Brig/Team/DB.hs

```diff
@@ -26,7 +26,7 @@ import Brig.Types.Common
 import Brig.Types.User
 import Brig.Types.Team.Invitation
 import Galley.Types.Teams (Role)
-import Cassandra
+import Cassandra (MonadClient, BatchType(BatchLogged), Consistency(Quorum), PrepQuery, R, W, addPrepQuery, batch, emptyPage, hasMore, liftClient, paginate, paginateC, params, paramsP, query1, result, retry, setConsistency, setType, write, x1, x5)
```

Comment:
This seems a smidge silly; I'm still in favour of being explicit in some way/shape/form, but qualified imports would probably be the way to go here. We really do need to decide on some standard for this tho 😋